benchmarks/distributed/rpc/rl/README.md

```diff
@@ -0,0 +1,64 @@
+# Distributed RPC Benchmark
+
+This tool is used to measure pytorch distributed rpc throughput and latency. This
+is helpful for evaluating the suitability of rpc to applications using pytorch distributed.
+
+In addition to printing measurements, it produces a JSON file.  Users may choose a single argument to provide multiple comma separated entries for (ie: `world_size="10,50,100"`) in which case the JSON file produced can be passed to the plotting repo to visually see how results differ.  In this case, each entry for the variable argument will be placed on the x axis.
+
+The benchmark results comprise of 4 key metrics:
+1. _Agent Latency_ - How long does it take from the time the first `select_action` request in a batch is received from an observer to the time an action is selected by the agent for each request in that batch.  If not using batch, you can think of it as `batch_size=1`.
+2. _Agent Throughput_ - The number of request processed per second for a given batch.  Is literally computed as `(batch_size / agent_latency)`.  If not using batch, you can think of it as `batch_size=1`.
```

Comment:
not sure if this is a typo?

batch.  Is literally computed as   --> batch, which is computed as ??

Reply:
Agent throughput is computed as batch_size/agent_latency.  What looks wrong to you?